src/identity_component.cairo

```diff
@@ -32,7 +32,7 @@ pub mod IdentityComponent {
     #[storage]
     pub struct Storage {
         Identity_execution_nonce: felt252,
-        Identity_keys: Map<felt252, Key>,
+        Identity_keys: Map<felt252, KeyDetails>,
         Identity_keys_by_purpose: Map<felt252, StorageArrayFelt252>,
         Identity_executions: Map<felt252, Execution>,
         Identity_claims: Map<felt252, Claim>,
```

Comment:
Isn't lowercase more suitable for storage variables?
Why capitalize the i in Identity?

Reply:
Ok! Still no `#[substorage(v1)]` :c